Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/RobotCameraPipeline.java

```diff
@@ -61,24 +70,30 @@ public Mat processFrame(Mat input) {
                 }
             }
 
-            // get the bounding rectangle around the largest contour
-            Rect boundingRect = Imgproc.boundingRect(contours.get(maxValIdx));
+            // crops contour if the area is below a certain size
+            if (maxVal >= Constants.CONTOUR_MIN_SIZE)
+            {
```

Comment:
Use the autoformat to fix all these types of formatting issues. https://google.github.io/styleguide/javaguide.html#s4.1-braces
```suggestion
            if (maxVal >= Constants.CONTOUR_MIN_SIZE) {
```

Reply:
Outdated